src/main/java/home/intexsoft/bank_application/command/Command.java

```diff
@@ -0,0 +1,113 @@
+package home.intexsoft.bank_application.command;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class Command {
+
+    private CommandType name;
+    private Map<CommandAttribute, String> attributes = new HashMap<>();
+
+    public CommandType getName() {
+        return name;
+    }
+
+    public void setName(CommandType name) {
+        this.name = name;
+    }
+
+    Map<CommandAttribute, String> getAttributes() {
+        return attributes;
+    }
+
+    public abstract void execute();
+
+    public enum CommandType {
+
+        BANKS("banks"),
+        CLIENTS("clients"),
+        OPERATIONS("operations"),
+        ADD_BANK("add bank"),
+        DELETE_BANK("delete bank"),
+        FIND_CLIENTS_OF_BANK("find clients of bank"),
+        VIEW_ALL_BANKS("view all banks"),
+        ADD_CLIENT("add client"),
+        DELETE_CLIENT("delete client"),
+        DELETE_ALL_CLIENTS_OF_BANK("delete all clients of bank"),
+        FIND_BANK_ACCOUNTS_OF_CLIENT("find bank accounts of client"),
+        ADD_BANK_ACCOUNT("add bank account"),
+        FIND_OPERATIONS_OF_CLIENT("find operations of client"),
+        ADD_CURRENCY("add currency"),
+        ADD_MONEY_TRANSFER("transfer money"),
+        ADD_SALARY_PAYMENT("pay salary");
+
+        private String commandName;
+
+        CommandType(String commandName) {
+            this.commandName = commandName;
+        }
+
+        public String getCommandName() {
+            return commandName;
+        }
+    }
+
+    public enum ClientStatusType {
+
+        INDIVIDUAL("individual"),
+        ENTITY("entity");
+
+        private String clientStatusName;
+
+        ClientStatusType(String clientStatusName) {
+            this.clientStatusName = clientStatusName;
+        }
+
+        public String getClientStatusName() {
+            return clientStatusName;
+        }
+
+        public void setClientStatusName(String clientStatusName) {
```

Comment:
@DmitryZagorski, do you really need getter here, do not create methods set b y default, remember about encapsulation.